backend/cli.py

```diff
@@ -47,17 +49,24 @@ def update_ids(ids: list[str]):
 
 
 @app.command()
-def index_meilisearch():
+def index_meilisearch(first_time: bool = False):
     if get_settings().app_environment == Environment.DEVELOPMENT:
         # Is ran at startup in production
         update_index()
 
+    db = database.SessionLocal()
+
+    if first_time:
+        db_media = get_all_media(db)
+    else:
+        db_media = get_recently_updated_media(db, hours=1)
```

Comment:
Should this be more than 1 hour now to be sure? Since we have to do `--first-time` every time?

Reply:
This is the else for when it isn't first time :p The other one takes everything

Reply:
True lol, but this will basically never be used then right? Other than locally

Reply:
Yup, it was made a "long" time ago when we didn't know any better :D But I still think it makes sense for the shorter flow. Might be useful later :shrug: